app/views/docs/graphql.phtml

```diff
@@ -0,0 +1,529 @@
+<!---->
+<!--<p>For example, given a project with a collection with ID <b>movies</b>, with attributes <b>name</b> and <b>length</b>, of types <b>string</b> and <b>integer</b> respectively; the following queries could be executed:</p>-->
+<!---->
+<!--<ul class="phases clear" data-ui-phases>-->
+<!--    <li>-->
+<!--        <h3>Create</h3>-->
+<!--        <div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">-->
+<!--            <pre class="line-numbers"><code class="prism language-graphql" data-prism>mutation CreateMovie($name: String!, $length: Int!) {-->
+<!--    moviesCreate(name: $name, length: $length) {-->
+<!--        _id-->
+<!--        name-->
+<!--        length-->
+<!--    }-->
+<!--}</code></pre>-->
+<!--        </div>-->
+<!--    </li>-->
+<!--    <li>-->
+<!--        <h3>List</h3>-->
+<!--        <div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">-->
+<!--            <pre class="line-numbers"><code class="prism language-graphql" data-prism>query ListMovies($limit: Int, $offset: Int, cursor: String, cursorDirection: String, orderAttributes: [String!], orderTypes: [String!]) {-->
+<!--    moviesList(limit: $limit, offset: $offset, cursor: $cursor, cursorDirection: $cursorDirection, orderAttributes: $orderAttributes, orderTypes: $orderType) {-->
+<!--        _id-->
+<!--        name-->
+<!--        length-->
+<!--    }-->
+<!--}</code></pre>-->
+<!--        </div>-->
+<!--    </li>-->
+<!--    <li>-->
+<!--        <h3>Get</h3>-->
+<!--        <div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">-->
+<!--            <pre class="line-numbers"><code class="prism language-graphql" data-prism>query GetMovie($id: String!) {-->
+<!--    moviesGet(id: $id) {-->
+<!--        _id-->
+<!--        name-->
+<!--        length-->
+<!--    }-->
+<!--}</code></pre>-->
+<!--        </div>-->
+<!--    </li>-->
+<!--    <li>-->
+<!--        <h3>Update</h3>-->
+<!--        <div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">-->
+<!--            <pre class="line-numbers"><code class="prism language-graphql" data-prism>mutation UpdateMovie {-->
+<!--    moviesUpdate(name: $name, length: $length) {-->
+<!--        _id-->
+<!--        name-->
+<!--        length-->
+<!--    }-->
+<!--}</code></pre>-->
+<!--        </div>-->
+<!--    </li>-->
+<!--    <li>-->
+<!--        <h3>Delete</h3>-->
+<!--        <div class="ide margin-bottom" data-lang="graphql" data-lang-label="GraphQL">-->
+<!--            <pre class="line-numbers"><code class="prism language-graphql" data-prism>mutation DeleteMovie($id: String!) {-->
+<!--    moviesDelete(id: $id)-->
+<!--}</code></pre>-->
+<!--        </div>-->
+<!--    </li>-->
+<!--</ul>-->
+
+<h3><a href="/docs/graphql#sdk-usage" id="sdk-usage">SDK Usage</a></h3>
+
+<p>The wide range of Appwrite SDKs also support GraphQL in addition to the REST services.</p>
```

Comment:
```suggestion
<p>Appwrite SDKs also support GraphQL in addition to the REST services.</p>
```

Again, no point in colorful language, people read docs to get answers not to see if they should use it. (I'm less strongly opinionated about this particular case.